backend/src/backend/tests.py

```diff
@@ -40,3 +40,22 @@ def test_list_tasks_unauthorized_user():
     response = client.get("/api/tasks")
     assert response.status_code == 401
     assert response.json() == {"detail": "User not logged in"}
+
+
+# Chat enpoint
+# Test for malformed request
```

Comment:
The test function names already contain the information, so these comments (lines 46 and 55) are redundant.